sormas-app/app/src/main/java/de/symeda/sormas/app/pathogentest/edit/PathogenTestEditActivity.java

```diff
@@ -84,7 +92,42 @@ public void saveData() {
 		}
 
 		final PathogenTest pathogenTestToSave = getStoredRootEntity();
+		final Case associatedCase = pathogenTestToSave.getSample().getAssociatedCase();
+
+		if (associatedCase != null) {
+			DiseaseVariant caseDiseaseVariant = DatabaseHelper.getDiseaseVariantDao().queryForId(associatedCase.getDiseaseVariant().getId());
+			DiseaseVariant newDiseaseVariant = pathogenTestToSave.getTestedDiseaseVariant();
+			if (pathogenTestToSave.getTestResult() == PathogenTestResultType.POSITIVE
+					&& pathogenTestToSave.getTestResultVerified() == true
+					&& !newDiseaseVariant.equals(caseDiseaseVariant)) {
+
+				String heading = I18nProperties.getCaption(Captions.caseUpdateCaseWithNewDiseaseVariant);
```

Comment:
Wrong captions referenced, app doesn't build because of that (please make sure that the app is building correctly)